### PR TITLE
Add the version to the Overview of the Admin Control panel

### PR DESCRIPTION
**Pull request by anirudhsudhir**
Solves issue #2373

This is my first open source contribution, so please do give me feedback regarding any mistakes in the code or contribution process.

### CODE REVIEW

File: apps/client/src/app/components/admin-overview/admin-overview.html [hunk 1]
```diff
@@ -3,6 +3,16 @@
     <div class="col">
       <mat-card appearance="outlined" class="mb-3">
         <mat-card-content>
+          <div class="d-flex my-3">
+            <div class="w-50" i18n>Version</div>
+            <div class="w-50">
+              <gf-value
```
Comment (dtslvr) [inline apps/client/src/app/components/admin-overview/admin-overview.html:9]: You don't need to use the value component (`gf-value`) here. Just use:
```
{{ version }}
```

**Review by dtslvr — CHANGES_REQUESTED** [review]
Congrats to your first PR @anirudhsudhir! 🙌🏻 

I have added one comment.

**Comment by anirudhsudhir** [comment]
Thank you!

I have made the necessary changes.

**Review by dtslvr — APPROVED** [review]
Thanks for the change!

**Pull request merged**